src/main/java/atoml/data/DataDescription.java

```diff
@@ -37,6 +37,11 @@
 	 * distribution used to generate numeric data
 	 */
 	private final AbstractRealDistribution distribution;
+	
+	/**
+	 * distribution used to generate numeric data for second class for separable data
+	 */
+	private final AbstractRealDistribution distribution2;
```

Comment:
please use a better name, e.g., distribution_separable or something similar that directly specifies the meaning of the attribute.